pkg/env/env.go

```diff
@@ -0,0 +1,13 @@
+package env
+
+const (
+	DefaultBinaryDir       = "/root/assets/testbin/"
+	EnvBinUrl              = "FIRMWARECI_BINURL"
+	EnvBinDir              = "FIRMWARECI_BINDIR"
+	EnvSSHHostPublic       = "FIRMWARECI_HOSTPUB"
+	EnvSSHPrivatePW        = "FIRMWARECI_SSHPRIVATEPW"
+	EnvS3Access            = "FIRMWARECI_S3_ACCESS"
+	EnvS3Secret            = "FIRMWARECI_S3_SECRET"
+	EnvS3Region            = "FIRMWARECI_S3_REGION"
+	FirmwareciSSHPublicKey = "/root/.shh/firmwareci.pub"
```

Comment:
still a type here. It's ssh and not shh

Reply:
Rip, I thought I had fixed that . Sorry about that...